spec/factories/user_submitted_source_records.rb

```diff
@@ -15,7 +15,7 @@
         {
           "id": "10.1016/j.petrol.2016.02.026",
           "type": "doi",
-          "url": "http://dx.doi.org/10.1016/j.petrol.2016.02.026",
+          "url": "https://dx.doi.org/10.1016/j.petrol.2016.02.026",
```

Comment:
I don't think there are any changes to the user submitted model spec...it is a pretty bare bones spec at the moment.